servo-aws/src/main/java/com/netflix/servo/publish/cloudwatch/CloudWatchMetricObserver.java

```diff
@@ -165,10 +183,19 @@ public void updateImpl(List<Metric> metrics) {
     }
 
     private void putMetricData(List<Metric> batch) {
+        DynamicCounter.increment(METRICS_COUNTER, batch.size());
+        final Stopwatch s = DynamicTimer.start(PUTS_TIMER);
```

Comment:
Same here. Are they dynamic monitors because you want to get the expiration behavior?